### PR TITLE
Updated piety policy tree

### LINKED ISSUE #4068: Some of social policies are different than in vanilla civ5
https://civilization.fandom.com/wiki/Social_policies_(Civ5)

### PR DESCRIPTION
**Pull request by xlenstra**
This PR updates the piety policy tree to G&K, continuing development of religion (see #4290) and finally finishing off development of policies (see #4084), thereby fixes #4068.
It also removes the space added to the "Patronage " and "Patronage  Complete" policies earlier for differentiating it from the then-recently removed "Patronage" policy.

Added uniques:
"[amount]% of excess happiness converted to [Stat]"

Deprecated uniques:
"50% of excess happiness added to culture towards policies"

### CODE REVIEW

**Comment by logicminimal** [comment]
what will happen if religion is disabled?

**Comment by xlenstra** [comment]
> what will happen if religion is disabled?

Nothing special, same as in the original

**Comment by logicminimal** [comment]
> > what will happen if religion is disabled?
> 
> Nothing special, same as in the original

what happens in the original?

**Comment by ravignir** [comment]
> 
> 
> > > what will happen if religion is disabled?
> > 
> > 
> > Nothing special, same as in the original
> 
> what happens in the original?

Piety branch is disabled i think.

File: core/src/com/unciv/models/ruleset/unit/BaseUnit.kt [hunk 1]
```diff
@@ -476,6 +476,7 @@ class BaseUnit : INamed, INonPerpetualConstruction, ICivilopediaText {
 
             "Nuclear Weapon" -> isNuclearWeapon()
             "Great Person", "Great" -> isGreatPerson()
+            "Religious", "religious" -> uniques.contains("Religious Unit")
```
Comment (yairm210) [inline core/src/com/unciv/models/ruleset/unit/BaseUnit.kt:479]: Please don't allow the lowercase version, why complicate?

**Comment by xlenstra** [comment]
> > > > what will happen if religion is disabled?
> > > 
> > > 
> > > Nothing special, same as in the original
> > 
> > 
> > what happens in the original?
> 
> Piety branch is disabled i think.

My test game basically allowed taking piety tree policies without any differences, even if they had no effect

**Pull request merged**